_pages/production.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Production deployment
 lead: >
-  Once you’ve tested your integration in [our sandbox environment](https://dashboard.int.identitysandbox.gov/), you can request deployment to the Login.gov production environment.
+  Once you’ve setup and tested your integration in [our dashboard](https://dashboard.int.identitysandbox.gov/), you can request deployment to the Login.gov production environment.
```

Comment:
[nitpick] `set up` (setup is a noun, set up is a verb)

Reply:
question -- is this confusing? The partner sets up their configuration in the partner dashboard, and then tests in their sandbox evironment. this feels like it implies that there is testing that is possible in the dashboard. (i know this isn't really a content change ticket, though)

Reply:
What about "Once you've set up your integration through our dashboard and tested..."? Or I'd say leave it for now and we deal with the sandbox problem across the site all at once. Somehow.